simulator/src/main/cpp/ChirpBluetoothPacket.h

```diff
@@ -0,0 +1,34 @@
+#define BLUETOOTH_INIT_PACKET 0
+#define BLUETOOTH_DATA_PACKET 1
+
+#define MAX_BLUETOOTH_DATA_SIZE 123
+
+/**
+ * Bluetooth Packet Format:
+ *      Type 1 -- Init Packet:
+ *          type(1byte->1char->int) groupId(1bytes->1char->1digit int {ie. 1,2,...,9}) nodeId(2bytes->2char->2digit int) /0
```

Comment:
Might remove this for now and add in if more startup information is required

Reply:
Ill remove the code to set the group id and node id from the packet but ill leave the packet type 0 code in the bluetooth packet class so we dont have to redo the serialize and parse functions if we want to add stuff in.

Reply:
just made the change in the router to use the bluetoothpacket nodeid and group for the radio packet

Reply:
These are required because we need the nodeid and groupid for routing before a person potentially send a data message. This is why on app boot we should send this init packet to setup the radio and router params